lib/antialiasing-comparator.js

```diff
@@ -80,7 +95,7 @@ module.exports = class AntialiasingComparator {
         // if either the darkest or the brightest pixel has more than 2 equal siblings in both images
         // (definitely not anti-aliased), this pixel is anti-aliased
         return (!this._isAntialiased(img1, minX, minY, data) && !this._isAntialiased(img2, minX, minY, data)) ||
-               (!this._isAntialiased(img1, maxX, maxY, data) && !this._isAntialiased(img2, maxX, maxY, data));
+            (!this._isAntialiased(img1, maxX, maxY, data) && !this._isAntialiased(img2, maxX, maxY, data));
     }
 
     _brightnessDelta(color1, color2) {
```

Comment:
I know it's not your code, but why not
```
_brightnessDelta(color1, color2) {
   const rgb2y = (r, g, b) => r * 0.29889531 + g * 0.58662247 + b * 0.11448223;
   return rgb2y(color1.R - color2.R, color1.G - color2.G, color1.B - color2.B);
}
```
I think 3 multiplications will be faster than 6. I'm not sure V8 can optimize such things.

If you have image `1000x1000`, you make extra `24 000 000` multiplications.

Reply:
I missed the fact, that antialiasing comparator is called only for different pixels, so boost will be very small.